plugin/vcs/vcs.go

```diff
@@ -79,6 +79,15 @@ type UserInfo struct {
 	State UserState `json:"state"`
 }
 
+// ProjectMember is the API message for project member info.
+type RepositoryMember struct {
+	Email   string             `json:"email"`
+	Name    string             `json:"name"`
+	State   UserState          `json:"state"`
+	Role    common.ProjectRole `json:"role"`
+	VCSRole string             `json:"vcs_role"`
```

Comment:
I am not sure if we need json annotation here since we don't persist it.

Reply:
we still need to unmarshal the JSON string fetched from the VCS. The annotation here is to make sure this process can go without a hitch.